packages/space-opera/src/components/poster_controls/poster_controls.ts

```diff
@@ -35,60 +37,86 @@ import {getCameraState} from '../model_viewer_preview/model_viewer_preview.js';
 /** Allow users to create / display a poster. */
 @customElement('me-poster-controls')
 export class PosterControlsElement extends ConnectedLitElement {
-  static styles = posterControlsStyles;
+  static styles = [posterControlsStyles, snackbarStyles];
 
   @internalProperty() poster?: string;
+  @internalProperty() className: string = '';
+  @internalProperty() snackBody: string = '';
+  @internalProperty() cameraSnippet: Camera = {};
 
   stateChanged(state: State) {
     this.poster = getConfig(state).poster;
+    this.cameraSnippet = getCamera(state);
   }
 
   render() {
     return html`
-      <me-expandable-tab tabName="Poster">
-        <div slot="content">
-          <mwc-button unelevated class="PosterButton" 
-            @click="${this.onCreatePoster}">Create Poster</mwc-button>
-          ${
+<me-expandable-tab tabName="Poster">
+  <div slot="content">
+    <me-card title="Poster Creation">
+      <div slot="content">
+        <mwc-button unelevated class="PosterButton" 
+          @click="${
+        this.onCreatePoster}">Generate Poster At Initial Camera</mwc-button>
+        <div class="PosterHelperButtons">
+        ${
     !!this.poster ? html`
-          <mwc-button unelevated class="PosterButton"
-            @click="${this.onDownloadPoster}">Download</mwc-button>
-          <mwc-button unelevated class="PosterButton"
-            @click="${this.onDisplayPoster}">Display Poster</mwc-button>
-          <mwc-button unelevated class="PosterButton"
-            @click="${this.onDeletePoster}">Delete Poster</mwc-button>` :
+        <mwc-button unelevated class="PosterButton"
+          @click="${this.onDownloadPoster}">Download Poster</mwc-button>
+        <mwc-button unelevated class="PosterButton"
+          @click="${this.onDisplayPoster}">Redisplay Poster</mwc-button>
```

Comment:
I think Display is better.

Reply:
done

Reply:
Also, when you display the poster, you need to reset the user's camera to the initial camera so that when they interact, the transition is seamless.

Reply:
gotcha, was gonna ask about that